web/app/templates/base.html

```diff
@@ -3,6 +3,7 @@
 <meta charset="UTF-8" />
 <meta name="google" content="notranslate" />
 <meta http-equiv="Content-Language" content="sv_SE" />
+<meta name="csrf-token" content="{{ csrf_token() }}">
```

Comment:
I've thought about deleting this element as soon as the config of jQuery's ajax is done. But the token will authorize every ajax-call so this wont protect us from XSS-attacks.